python/python/lancedb/embeddings/bedrock.py

```
@@ -15,6 +15,417 @@
+        }
+
+        response = self.client.invoke_model(
+            modelId="cohere.embed-multilingual-v3",
+            body=json.dumps(request_body)
+        )
+
+        response_body = json.loads(response.get("body").read())
+        embeddings = response_body.get("embeddings", [])
+        return np.array(embeddings)
+
+    def sanitize_input(self, images: IMAGES) -> Union[List[bytes], np.ndarray]:
```

Ruff (0.8.2)

Undefined name `IMAGES`

(F821)